src/azure-cli/azure/cli/command_modules/cosmosdb/custom.py

```diff
@@ -549,6 +556,109 @@ def _populate_sql_container_definition(sql_container_resource,
     return True
 
 
+def _validate_and_populate_client_encryption_policy(sql_container_resource, client_encryption_policy):
+    if client_encryption_policy is not None:
+        from azure.cli.core.util import shell_safe_json_parse
+        data = shell_safe_json_parse(json.dumps(client_encryption_policy))
+
+        if "includedPaths" in data:
+            includedPaths = data['includedPaths']
+        else:
+            raise CLIError(None, "includedPaths missing in Client Encryption Policy. Please verify your Client Encryption Policy JSON string")
```

Comment:
Please use the error type defined in`azure.cli.core.azclierror`, eg, `InvalidArgumentValueError`

Reply:
Sure.